components/Article/AuthorList/Author/__tests__/index.test.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { AppProps } from '../../../../../types';
+import { LocaleProvider } from '../../../../../providers/localeProvider';
+import messages from '../../../../../i18n/locales/en.json';
+import Author from '..';
+
+const i18nData = {
```

Comment:
We shouldn't pass i18n data to unit tests. We shouldn't add translations/English text to unit tests and only render the translation keys.

Otherwise, we need to update the snapshots every time the translations get updated.

If we want to test the actual translations and fitting with the components, see the visual regression tests issue.